networkx/utils/tests/test_misc.py

```diff
@@ -127,15 +127,15 @@ def test_make_str_with_unicode():
 class TestNumpyArray:
     @classmethod
     def setup_class(cls):
-        global numpy
+        global np
         global assert_allclose
-        numpy = pytest.importorskip("numpy")
-        assert_allclose = numpy.testing.assert_allclose
+        np = pytest.importorskip("numpy")
+        assert_allclose = np.testing.assert_allclose
```

Comment:
This seems to deviate from the pattern established in the rest of the PR.

Reply:
I went with this pattern everywhere (it is consistent with the np import policy everywhere else.

Reply:
Ah sorry, I wasn't referring to the aliasing but the `assert_allclose = np.testing.assert_allclose`. I was thinking `from numpy.testing import assert_allclose`, but it's just a nit and fine as is.